monkey/infection_monkey/credential_collectors/mimikatz_collector/mimikatz_credential_collector.py

```diff
@@ -11,12 +13,32 @@
 logger = logging.getLogger(__name__)
 
 
+MIMIKATZ_CREDENTIAL_COLLECTOR_TAG = "MimikatzCredentialsStolen"
+T1003_ATTACK_TECHNIQUE_TAG = "T1003"
+T1005_ATTACK_TECHNIQUE_TAG = "T1005"
```

Comment:
Tags should be lowercase. Attack techniques should begin with "attack". Example: "attack-t1003"